stdlib/2/typing.pyi

```diff
@@ -16,6 +16,7 @@ Type = object()
 _promote = object()
 no_type_check = object()
 ClassVar = object()
+Protocol = object()
```

Comment:
It's special either way: if I do `Protocol = object()` elsewhere, mypy probably won't let me do `class SupportsFloat(Protocol): pass`. Using a class is at least a little closer to what Protocol actually does at runtime, even if Protocol obviously isn't a normal class.